deploy/charts/library/templates/_cluster-psp.tpl

```diff
@@ -2,6 +2,8 @@
 RoleBindings needed to enable Pod Security Policies for a CephCluster.
 */}}
 {{- define "library.cluster.psp.rolebindings" }}
+{{- if semverCompare "<1.25.0-0" .Capabilities.KubeVersion.GitVersion }}
+---
 apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
@travisn The condition here means that anything importing this will only have something to import if the semverCompare also allows it. I tested manually, and all is good.